test/pummel/test-regress-GH-814_2.js

```diff
@@ -79,14 +79,14 @@ function writer() {
         bufPool.length = 0;
       }
       process.nextTick(writer);
-      //console.error('o');
+      // console.error('o');
     }
   }
 
 }
 
 function writerCB(err, written) {
-  //console.error('cb.');
+  // console.error('cb.');
```

Comment:
Suggestion: remove this. This is not necessary in the current `ifError` implementation anymore as the current stack is going to be included as well.